Mathlib/SetTheory/Ordinal/NaturalOps.lean

```diff
@@ -619,50 +606,41 @@ theorem lt_nmul_iff₃ :
     simpa only [nadd_assoc]
   · rintro ⟨a', ha, b', hb, c', hc, h⟩
     have := h.trans_lt (nmul_nadd_lt₃ ha hb hc)
-    repeat' rw [nadd_lt_nadd_iff_right] at this
+    repeat rw [nadd_lt_nadd_iff_right] at this
     assumption
 
-theorem nmul_le_iff₃ :
-    a ⨳ b ⨳ c ≤ d ↔
-      ∀ a' < a, ∀ b' < b, ∀ c' < c,
-        a' ⨳ b ⨳ c ♯ a ⨳ b' ⨳ c ♯ a ⨳ b ⨳ c' ♯ a' ⨳ b' ⨳ c' <
-          d ♯ a' ⨳ b' ⨳ c ♯ a' ⨳ b ⨳ c' ♯ a ⨳ b' ⨳ c' := by
-  rw [← not_iff_not]; simp [lt_nmul_iff₃]
-
-theorem lt_nmul_iff₃' :
-    d < a ⨳ (b ⨳ c) ↔
-      ∃ a' < a, ∃ b' < b, ∃ c' < c,
-        d ♯ a' ⨳ (b' ⨳ c) ♯ a' ⨳ (b ⨳ c') ♯ a ⨳ (b' ⨳ c') ≤
-          a' ⨳ (b ⨳ c) ♯ a ⨳ (b' ⨳ c) ♯ a ⨳ (b ⨳ c') ♯ a' ⨳ (b' ⨳ c') := by
-  simp only [nmul_comm _ (_ ⨳ _), lt_nmul_iff₃, nadd_eq_add, NatOrdinal.toOrdinal_toNatOrdinal]
+theorem nmul_le_iff₃ : a ⨳ b ⨳ c ≤ d ↔ ∀ a' < a, ∀ b' < b, ∀ c' < c,
+    a' ⨳ b ⨳ c ♯ a ⨳ b' ⨳ c ♯ a ⨳ b ⨳ c' ♯ a' ⨳ b' ⨳ c' <
+      d ♯ a' ⨳ b' ⨳ c ♯ a' ⨳ b ⨳ c' ♯ a ⨳ b' ⨳ c' := by
+  rw [← not_iff_not]
+  simp [lt_nmul_iff₃]
+
+@[deprecated lt_nmul_iff₃ (since := "2024-11-19")]
+theorem lt_nmul_iff₃' : d < a ⨳ (b ⨳ c) ↔ ∃ a' < a, ∃ b' < b, ∃ c' < c,
+    d ♯ a' ⨳ (b' ⨳ c) ♯ a' ⨳ (b ⨳ c') ♯ a ⨳ (b' ⨳ c') ≤
+      a' ⨳ (b ⨳ c) ♯ a ⨳ (b' ⨳ c) ♯ a ⨳ (b ⨳ c') ♯ a' ⨳ (b' ⨳ c') := by
+  simp only [nmul_comm _ (_ ⨳ _), lt_nmul_iff₃, nadd_eq_add, toOrdinal_toNatOrdinal]
   constructor <;> rintro ⟨b', hb, c', hc, a', ha, h⟩
   · use a', ha, b', hb, c', hc; convert h using 1 <;> abel_nf
   · use c', hc, a', ha, b', hb; convert h using 1 <;> abel_nf
 
-theorem nmul_le_iff₃' :
-    a ⨳ (b ⨳ c) ≤ d ↔
-      ∀ a' < a, ∀ b' < b, ∀ c' < c,
-        a' ⨳ (b ⨳ c) ♯ a ⨳ (b' ⨳ c) ♯ a ⨳ (b ⨳ c') ♯ a' ⨳ (b' ⨳ c') <
-          d ♯ a' ⨳ (b' ⨳ c) ♯ a' ⨳ (b ⨳ c') ♯ a ⨳ (b' ⨳ c') := by
-  rw [← not_iff_not]; simp [lt_nmul_iff₃']
+private theorem nmul_le_iff₃' : a ⨳ (b ⨳ c) ≤ d ↔ ∀ a' < a, ∀ b' < b, ∀ c' < c,
```

Comment:
You can use this to prove `lt_nmul_iff₃'`, if only to not increase lines of code / duplication.